policyengine_taxsim/config/variable_mappings.yaml

```diff
@@ -522,6 +651,10 @@ policyengine_to_taxsim:
     group_order: 4
     full_text_group: "State Tax Calculation"
     group_column: 1
+    special_cases:
+      - ms:
+          implemented: false
+          variable: na_pe
```

Comment:
Why would we have special case code here?

Reply:
it gives exception while call `ms_eitc` variable